src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1333,7 +1333,7 @@ public static function getMimeTypes(string $format)
     public function getFormat(?string $mimeType)
```

Comment:
Does not make sense to me that the mimeType can be null.

Reply:
I agree. Maybe we can deprecate passing `null` in 5.4.